api/proto/teleport/legacy/types/events/events.proto

```diff
@@ -3562,10 +3571,17 @@ message LockCreate {
   ];
 
   // Target describes the set of interactions that the lock applies to
+  // Deprecated: use Lock instead.
```

Comment:
1. Should we add a "DELETE IN vX" comment so we follow up on this later?

2. Should we set both fields for the time being, for backwards compatibility? It's not super relevant but it could play a part in eventual rollbacks.

Reply:
1. Hmm, if we delete a field won't that prevent new clients from reading old events in the future?
2. That's not a bad idea, I'll do that

Reply:
1. True, we should keep this around and probably move from the old to new field so most of the code doesn't need to care.